### PR TITLE
feat(blog): Add post about Gatsby team at DrupalCon and JAMstack Conf

### PR DESCRIPTION
**Pull request by prestonso**
## Description

Wrote a super quick blog post to highlight Gatsby folks spreading across the country this week! Approved by @stephen-gatsby 

/cc @marcysutton @amberleyromo @marisamorby @DSchau @lindaleebumblebee 

## Related Issues

None

### CODE REVIEW

File: docs/blog/2019-04-08-meet-the-gatsby-team-at-drupalcon-seattle-and-jamstack-conf-nyc/index.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+“As designers and developers of software, we act as gatekeepers in people’s lives more than we realize,” states the keynote abstract. “Now more than ever, it is our duty to create digital experiences that honor our users’ rights to inclusion–no matter where they live, or how they access the Web. This applies to both authors and contributors, as well as end-users and consumers; we must do more to protect everyone’s rights to access, safety, security, and privacy.”
+
+## Meet Gatsby CEO Kyle Mathews at Decoupled Summit
+
+Tomorrow, our CEO **[Kyle Mathews](https://twitter.com/kylemathews)** will be joined by [yours truly](https://twitter.com/prestonso) on stage at DrupalCon’s [Decoupled Summit](https://events.drupal.org/seattle2019/decoupled-summit) to talk about how Gatsby helps teams and organizations around the world build blazing fast web applications, all while having fun in the process. Catch our presentation during the Decoupled Summit on Tuesday from 11:15am–12pm.
```
Comment (DSchau) [inline docs/blog/2019-04-08-meet-the-gatsby-team-at-drupalcon-seattle-and-jamstack-conf-nyc/index.md]:
```suggestion
Tomorrow (April 9), our CEO **[Kyle Mathews](https://twitter.com/kylemathews)** will be joined by [yours truly](https://twitter.com/prestonso) on stage at DrupalCon’s [Decoupled Summit](https://events.drupal.org/seattle2019/decoupled-summit) to talk about how Gatsby helps teams and organizations around the world build blazing fast web applications, all while having fun in the process. Catch our presentation during the Decoupled Summit on Tuesday from 11:15am–12pm.
```

**Comment by jlengstorf** [comment]
@prestonso I'm also at [JS Heroes](https://jsheroes.io/) this week doing a [workshop](https://jsheroes.io/workshops/react) and a [talk](https://jsheroes.io/#schedule). 😂

File: docs/blog/2019-04-08-meet-the-gatsby-team-at-drupalcon-seattle-and-jamstack-conf-nyc/index.md (hunk 1 above)
Comment (prestonso) [reply docs/blog/2019-04-08-meet-the-gatsby-team-at-drupalcon-seattle-and-jamstack-conf-nyc/index.md]: Thanks! Resolving now.

**Comment by prestonso** [comment]
@jlengstorf Adding your speaking to the content!

**Comment by jlengstorf** [comment]
Thanks @prestonso! Looks great!

**Comment by marcysutton** [comment]
Love it! Thanks for writing, @prestonso!

**Comment by prestonso** [comment]
Thanks all, merged!

**Comment by KyleAMathews** [comment]
Thanks @prestonso!

**Pull request merged**